### PR TITLE
(test) O3-2936 : add e2e  tests for edit and delete vitals and biometrics

### PR DESCRIPTION
**Pull request by senthil-athiban**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary
End-to-End tests for editing and deleting vitals.

## Related Issue
[O3-2936](https://openmrs.atlassian.net/browse/O3-2936)

[O3-2936]: https://openmrs.atlassian.net/browse/O3-2936?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by brandones** [comment]
@senthil-athiban Please fix the merge conflicts.

Sorry about the delay with a review. I have requested a review from @kdaud (or @jayasanka-sack if Daud cannot).

File: e2e/specs/biometrics.spec.ts [hunk 1]
```diff
@@ -64,6 +63,72 @@ test('Record biometrics', async ({ page }) => {
...
+
+  await test.step('And I click the `Edit` button', async () => {
+    await biometricsPage.page.getByRole('menuitem', { name: /edit/i }).click();
+  });
+
+  await test.step('Then I should see the biomterics form launch in the workspace in edit mode', async () => {
+    await expect(biometricsPage.page.getByText(/Edit vitals and biometrics/i)).toBeVisible();
+  });
+
+  await test.step('And I change the `Weight`', async () => {
```
Comment (kdaud) [inline e2e/specs/biometrics.spec.ts:82]:
```suggestion
  await test.step('When I change the weight to `70`', async () => {
```

Comment (kdaud) [reply e2e/specs/biometrics.spec.ts:82]: Let's consider revising at least two biometrics.

File: e2e/specs/vitals.spec.ts [hunk 2]
```diff
@@ -81,6 +81,79 @@ test('Record vital signs', async ({ page }) => {
...
+
+  await test.step('And I click the `Edit` button', async () => {
+    await vitalsPage.page.getByRole('menuitem', { name: /edit/i }).click();
+  });
+
+  await test.step('Then I should see the vitals form launch in the workspace in edit mode', async () => {
+    await expect(vitalsPage.page.getByText(/Edit vitals and biometrics/i)).toBeVisible();
+  });
+
+  await test.step('And I change the `Temperature`', async () => {
```
Comment (kdaud) [inline e2e/specs/vitals.spec.ts:100]:
```suggestion
  await test.step('When I change the temperature to `42`', async () => {
```

Comment (kdaud) [reply e2e/specs/vitals.spec.ts:100]: Let's consider revising at least two vitals.

**Review by kdaud — COMMENTED** [review]
Thanks @senthil-athiban. I've left few comments for you to address.

**Comment by kdaud** [comment]
@senthil-athiban fix the failing build.

**Comment by brandones** [comment]
Ping @senthil-athiban

**Comment by senthil-athiban** [comment]
@brandones as we have reverted the edit and delete feature for vitals and biometrics, can we close this one.?

**Pull request closed without merging**